ui/templates/homepage/partials/navbar.html

```diff
@@ -26,6 +26,11 @@
 							 <li class="nav-item">
 								<a class="nav-link active" aria-current="page" href="{% url 'get_csv' %}">My reservations</a>
 							</li>
+							 {% if user.is_superuser %}
+							 <li class="nav-item">
```

Comment:
If the user must be a superuser, you can move this to admin on the Movies page as it has no use to be on the page dedicated to the customers